### PR TITLE
Contact form modal

### PR DESCRIPTION
**Pull request by jennetturkkan**
Added: 
* Modal that appears after user submits message in contact form

Screen Shot 2020-11-18 at 19 43 47

### CODE REVIEW

**Review by hiba-machfej — APPROVED** [review]

File: src/components/ContactForm/index.jsx [hunk 1]
```diff
@@ -7,11 +7,12 @@ import firebase from "../../firebaseConfig";
 import { getErrorClass } from "../../utils/formErrorHelpers";
 import "./index.scss";
 
-export default function ContactForm() {
+export default function ContactForm({ handleShow }) {
   const { register, handleSubmit, errors, reset } = useForm();
   const { t } = useTranslation();
 
   const onSubmit = (data) => {
+    handleShow();
```
Comment (Jamil-Bailony) [inline src/components/ContactForm/index.jsx:15]: I think you should first check if the message has been saved successfully and then eaither say that "thank you ...." or "Oops! something happend try again ...". you can use catch for error I think

**Review by Jamil-Bailony — COMMENTED** [review]
just mention about where to place the modalShow function

File: src/components/ContactForm/index.jsx (hunk 1 above)
Comment (jennetturkkan) [reply src/components/ContactForm/index.jsx:15]: it won't come to submit stage, so I think we don't need it 
Screen Shot 2020-11-18 at 22 19 27

Comment (jennetturkkan) [reply src/components/ContactForm/index.jsx:15]: and if it's sent to firebase, then it will be saved

**Comment by jennetturkkan** [comment]
> just mention about where to place the modalShow function

what do you mean?

**Review by louisrli — APPROVED** [review]

**Comment by jennetturkkan** [comment]
@Ammar-64 can you merge please?

**Pull request merged**